src/main/java/com/internet/shop/Application.java

```diff
@@ -16,34 +16,29 @@
     private static final Injector injector = Injector.getInstance("com.internet.shop");
 
     public static void main(String[] args) {
-        final ProductService productService = (ProductService)
-                injector.getInstance(ProductService.class);
+        final ProductDaoJdbcImpl productDaoJdbcImpl = new ProductDaoJdbcImpl();
         final ShoppingCartService shoppingCartService =
                 (ShoppingCartService) injector.getInstance(ShoppingCartService.class);
         final UserService userService = (UserService) injector.getInstance(UserService.class);
         final OrderService orderService = (OrderService) injector.getInstance(OrderService.class);
 
-        Product banana = new Product("banana", 10.1);
-        Product orange = new Product("orange", 4.1);
-        Product lemon = new Product("lemon", 6.3);
-
-        productService.create(banana);
-        productService.create(orange);
-        productService.create(lemon);
+        final Product banana = new Product("banana", 10.1);
+        final Product orange = new Product("orange", 4.1);
+        final Product lemon = new Product("lemon", 6.3);
```

Comment:
Does the code in Application work? I don't see where you add products to the database using `productDaoJdbcImpl.create(product)`, but maybe you already have products in the DB. It might be a good idea to test if the `create()` method works well, though)

Reply:
Thank you, missed that create statement, true, well it can't work without it)